src/Test.UnitTests.Sarif/Core/WebRequestTests.cs

```diff
@@ -184,5 +185,27 @@ public void WebRequest_TryParse_ReturnsFalseOnInvalidRequest()
             succeeded.Should().BeFalse();
             webRequest.Should().BeNull();
         }
+
+        [Fact]
+        public void WebRequest_TryParse_HasAcceptablePerformance()
+        {
+            const string RequestString = @"GET /getSomethings?FirstName=test&LastName=test&AddressLine1=555%20110th%20Ave%20NE&City=Bellevue&Country=USA&PostalCode=98004&EmailAddress=test@somedomain.com&BusinessPhone=12345678901&MobilePhone=1234567890&AddressLine2=&AddressLine3 HTTP/1.1
```

Comment:
>RequestString [](start = 25, length = 13)

A sanitized version of the actual customer web request that exposed the problem.